riscv/mmu.cc

```diff
@@ -224,39 +225,41 @@ void mmu_t::load_slow_path_intrapage(reg_t len, uint8_t* bytes, mem_access_info_
 void mmu_t::load_slow_path(reg_t addr, reg_t len, uint8_t* bytes, xlate_flags_t xlate_flags)
```

Comment:
I think there is too much risk of someone accidentally using the short name `addr` instead of using `transformed_addr` within the body of this function.  So, maybe rename `addr` to `original_addr` or something?

Same comment applies to `store_slow_path`.

Reply:
I added https://github.com/riscv-software-src/riscv-isa-sim/pull/1718/commits/04eb65437bf3f6c7a32b7a276a46a74c8cc3bba6 and https://github.com/riscv-software-src/riscv-isa-sim/pull/1718/commits/4a4ab850de34a99489ac9c47702ef815e13b6889 to rename the function parameter ```addr``` to ```original_addr``` in ```load_slow_path``` and ```store_slow_path```.

I don't know if we should also rename the local variable ```addr``` in ```load_slow_path_intrapage``` and ```store_slow_path_intrapage```. (Additionally, the variable ```vaddr``` in ```mem_access_info_t```.)